test/integration/02-user-wallet/02-send-onchain.spec.ts

```diff
@@ -343,6 +345,9 @@ describe("UserWallet - onChainPay", () => {
       { $group: { _id: null, outgoingSats: { $sum: "$debit" } } },
     ])
     const { outgoingSats } = result || { outgoingSats: 0 }
+
+    // FIXME: 'withdrawalLimit' no longer exists as of
```

Comment:
so the test is passing because a negative amount throws an exception? 

we need to add the description or type to `.toThrow()` and fix te test

Reply:
I fixed this particular `FIXME` (https://github.com/GaloyMoney/galoy/pull/393/commits/0e48a4cdba5c3934716ae98b6209da4b146407f5) but I suspect the test is still broken because just a few lines above the `userWallet0.accountPath` seems to not exist on that object for example.

This test-fix is probably outside the scope of this PR though... or should I try to fix it here?

Reply:
https://github.com/GaloyMoney/galoy/blob/main/src/schema.ts#L259

Reply:
Ah ok my bad! I was going off a `console.warn` printed value of the `userWallet0` object in the tests lol

Reply:
Ok, I found it, we need to add +1 and use TransactionRestrictedError 

```js
import { TransactionRestrictedError } from "src/error"

....

const amount = userLimits.withdrawalLimit - outgoingSats + 1

await expect(userWallet0.onChainPay({ address, amount })).rejects.toThrow(TransactionRestrictedError)
```